services/sts/sts-core/src/test/java/org/apache/cxf/sts/operation/ValidateJWTTransformationTest.java

```diff
@@ -206,7 +206,9 @@ public void testJWTToSAMLTransformationRealm() throws Exception {
         TokenProviderResponse providerResponse = createJWT();
         Element wrapper = createTokenWrapper((String)providerResponse.getToken());
         Document doc = wrapper.getOwnerDocument();
-        wrapper = (Element)doc.appendChild(wrapper);
+        if (doc.getDocumentElement() == null) {
```

Comment:
@kaiyaok2 this change is a bit suspicious, why the document element will be null here?

Reply:
@kaiyaok2 I think this test should be refactored to create a new document (vs manipulating empty one which is static):

```
    private Element createTokenWrapper(String token) {
        Document doc = DOMUtils.createDocument();
        Element tokenWrapper = doc.createElementNS(null, "TokenWrapper");
        tokenWrapper.setTextContent(token);
        return tokenWrapper;
    }

```

Could you please make a change? Thank you.